intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -188,6 +187,12 @@ \subsection{Pre-defined user directories}
 We ellaborate more on the specific function of these locations in the following
 sections.
 
+\ifgent
+Note: \verb|$VSC_SCRATCH_KYUKON| and \verb|$VSC_SCRATCH| are the same directories.
```

Comment:
add: `("kyukon" is the name of the storage cluster where the default shared scratch filesystem is hosted).`